crypto/rand/drbg_lib.c

```diff
@@ -389,7 +448,23 @@ int RAND_DRBG_uninstantiate(RAND_DRBG *drbg)
      * initial values.
      */
     drbg->meth->uninstantiate(drbg);
-    return RAND_DRBG_set(drbg, drbg->type, drbg->flags);
+
+    /* The reset uses the default values for type and flags */
```

Comment:
Why?

Reply:
The commit comment explains why it does this..

Reply:
The commit message doesn't explain it, it's rather short.

The effect of this is if you did a RAND_DRBG_set(), and you uninstantiate it, it gets overridden by defaults now, which might be unexpected.